### PR TITLE
Admin server announcement uses new announcement span

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

The current admin announce is just notice span text, which is easily missed mixed in with all the other white noise of the chat box. Currently admins have to fill it with linebreaks or manually add their own spans to increase visibility.

This updates the admin announcement proc to use the new alert box divs, similar to other announcements, making it more visible.

## Why It's Good For The Game

Admin server-wide announcements are generally things you want the players to notice

image

## Changelog

:cl: LT3
admin: Server wide admin announcements now use an alert box like other announcements
/:cl:

### CODE REVIEW

File: code/__HELPERS/priority_announce.dm [hunk 1]
```diff
@@ -36,8 +36,9 @@
  * * players - a list of all players to send the message to. defaults to all players (not including new players)
  * * encode_title - if TRUE, the title will be HTML encoded
  * * encode_text - if TRUE, the text will be HTML encoded
+ * * send_to_world - if TRUE will send the announcement to the world rather than a list of players
  */
-/proc/priority_announce(text, title = "", sound, type, sender_override, has_important_message = FALSE, list/mob/players = GLOB.player_list, encode_title = TRUE, encode_text = TRUE, color_override)
+/proc/priority_announce(text, title = "", sound, type, sender_override, has_important_message = FALSE, list/mob/players = GLOB.player_list, encode_title = TRUE, encode_text = TRUE, color_override, send_to_world = FALSE)
```
Comment (MrMelbert) [inline code/__HELPERS/priority_announce.dm:41]: I don't like this parameter, and I don't necessarily like using this proc for out-of-ic announcements

Case in point: In your current implementation, you're sending admin announces to newscasters!

If you want the formatting it offers specifically, we should separate the formatting part into a separate proc.

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Pictures would be appreciated

File: code/__HELPERS/priority_announce.dm (hunk 1 above)
Comment (lessthnthree) [reply code/__HELPERS/priority_announce.dm:41]: I was thinking whether to split it or not, so yeah I'll look into that.

Comment (ZephyrTFA) [reply code/__HELPERS/priority_announce.dm:41]: can we just check for players being set to world

Comment (MrMelbert) [reply code/__HELPERS/priority_announce.dm:41]: `players = world` is an option but I would still prefer this proc stay for IC stuff so we don't accidentally leak stuff to newscasters (it's already happened once and almost happened twice)

**Review by Ghommie — APPROVED** [review]
Definitely a visibility improvement here.

File: code/__HELPERS/announcements.dm [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+	if(encode_title && title && length(title) > 0)
+		title = html_encode(title)
+		if(encode_text)
+			text = html_encode(text)
+			if(!length(text))
+				return
+
+		if(isnull(sender_override))
+			sender_override = "Server Announcement"
```
Comment (MrMelbert) [inline code/__HELPERS/announcements.dm:38]: IMO, for parity with the old formatting 
```suggestion
			sender_override = "Admin Announcement"
```
```suggestion
			sender_override = "Server Admin Announcement"
```

**Review by MrMelbert — COMMENTED** [review]
I'm a little worried that people confuse these for IC announcements now since they are formatted the same, but given the current form is not much better in that regard it's probably w/e

File: code/__HELPERS/announcements.dm [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+				return
+
+		if(isnull(sender_override))
+			sender_override = "Server Admin Announcement"
+
+		if(isnull(sound_override)) // this is not an IC announcement, so we don't adjust for announcer station traits
+			sound_override = 'sound/ai/default/attention.ogg'
+
+		if(isnull(color_override))
+			color_override = "grey"
```
Comment (Jacquerel) [inline code/__HELPERS/announcements.dm:44]: It'll make for a long signature but I don't see why these couldn't just be default values for these arguments, like the booleans already have

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Pull request merged**